myuw/static/css/typography.less

```diff
@@ -75,6 +75,19 @@ abbr[title] {
     font-size: @fontsize-small;
 }
 
+/** Can be used on an element or as a container for multiples **/
+.myuw-muted {
+    color: #777;
+
+    a&,
```

Comment:
I had to look this one up as I'd never seen this SASS/LESS syntax. But even now somewhat more informed, I'm not clear what you're trying to do. This compiles to `a.myuw-muted`  and `.myuw-muted a`, which both get the hover and visited style treatment you specified. But in the Welcome box only the later structure exists. The footer has `a.myuw-muted`, but not sure why you'd want to us that style in the footer where the links are white... Moreover, the Welcome links are blue by default...is that your intention?

Reply:
What I was going for here is the ability to either use .myuw-muted as a class you could add to any text or link which would make it a muted gray. 

Additionally, I added the "&" stuff in there (which compiles to 'a.myuw-muted' and '.myuw-muted a') so that you could use the .myuw-muted as a wrapper class over many items. I figured that would be more efficient for when you want to have a collection of muted text/links so you don't have to add the class to each text/link.

I didn't intend to add the myuw-muted class to the link in the footer, so good catch there. 

I think links, in general, are blue by default. The myuw-muted class is intended to overwrite that to make them gray. Here's what I see on my build, which was my intent:
![myuw__home](https://user-images.githubusercontent.com/6953502/32742601-cd169bf2-c85e-11e7-8774-495d6914a701.jpg)

Reply:
got it, that rationale makes sense to me.